integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+                'highway': u'secondary',
+                'name': u'არტემ ბალახაშვილის ქუჩა (Artem Balakhashvili St)',
+                'name:en': u'Artem Balakhashvili St',
+                'name:ka': u'არტემ ბალახაშვილის ქუჩა',
+                'oneway': u'yes',
+                'ref': u'შ 1',
+                'source': u'openstreetmap.org',
+                'surface': u'paved',
+            }),
+            dsl.relation(1, {
+                'is_in:country': u'Georgia',
+                'name': u'Batumi–Akhaltsikhe Highway',
+                'nat_ref': u'შ 1',
+                'network': u'ge:national',
+                'ref': u'შ 1',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[250762262]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 250762262,
+                'network': u'GE:national',
+            })
+
+    def test_grprovincial22(self):
+        import dsl
+
+        z, x, y = (16, 36383, 24903)
+
+        self.generate_fixtures(
+            dsl.is_in('GR', z, x, y),
+            # https://www.openstreetmap.org/way/503900908
+            dsl.way(503900908, dsl.tile_diagonal(z, x, y), {
+                'highway': u'secondary',
+                'oneway': u'yes',
+                'reg_ref': u'ΕΠ2;ΕΠ16',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'network': u'gr:provincial:22',
+                'ref': u'2202',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[503900908]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 503900908,
+                'network': u'GR:provincial:22',
+            })
+
+    def test_grprovincial52(self):
+        import dsl
+
+        z, x, y = (16, 37161, 24536)
+
+        self.generate_fixtures(
+            dsl.is_in('GR', z, x, y),
+            # https://www.openstreetmap.org/way/25144610
+            dsl.way(25144610, dsl.tile_diagonal(z, x, y), {
+                'highway': u'secondary',
+                'int_name': u'Drama - Mavrolefki',
+                'lanes': u'2',
+                'name': u'Αμφίπολη - Δράμα',
+                'oneway': u'yes',
+                'reg_ref': u'ΕΠ1',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'name': u'Δράμα - Μαυρολεύκη - Νέα Μπάφρα',
+                'network': u'gr:provincial:52',
+                'ref': u'5201',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[25144610]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 25144610,
+                'network': u'GR:provincial:52',
```

Comment:
We omitted the province code in France in the network value, should we here as well? Else please provide list of province numbers so the client-side code knows what to deal with as a look up instead of a regEx.

Reply:
Also, is the `52` part of the ref actually used in the wild on road signage or is it just `1` or `01`?

Reply:
According to [this OSM wiki page](https://wiki.openstreetmap.org/wiki/WikiProject_Greece/Provincial_Road_Network) there aren't any provincial signs. I did a few spot-checks, and wasn't able to find any, so I can't verify whether "5201" or "ΕΠ01" would be more appropriate to show.

Reply:
Removed the province code in dc9ca6b.